python/lsst/afw/math/_warpExposure.cc

```diff
@@ -0,0 +1,190 @@
+    wrappers.wrap([](auto &mod) {
+        mod.def("warpExposure", &warpExposure<DestExposureT, SrcExposureT>, "destExposure"_a, "srcExposure"_a,
+                "control"_a, "padValue"_a = edgePixel<DestMaskedImageT>(
+                        typename image::detail::image_traits<DestMaskedImageT>::image_category()));
+
+    });
+    declareImageWarpingFunctions<DestImageT, SrcImageT>(wrappers);
+    declareImageWarpingFunctions<DestMaskedImageT, SrcMaskedImageT>(wrappers);
+}
+
+void declareWarpExposure(lsst::utils::python::WrapperCollection &wrappers) {
+    using PyClass=py::class_<WarpingControl, std::shared_ptr<WarpingControl>>;
+    wrappers.wrapType(PyClass(wrappers.module,"WarpingControl"),
+            [](auto &mod,auto clsWarpingControl) {
+                clsWarpingControl.def(py::init<std::string, std::string, int, int, image::MaskPixel>(),
+                                      "warpingKernelName"_a, "maskWarpingKernelName"_a = "", "cacheSize"_a = 0,
+                                      "interpLength"_a = 0, "growFullMask"_a = 0);
+
+                /* Operators */
+
+                clsWarpingControl.def("getCacheSize", &WarpingControl::getCacheSize);
+                clsWarpingControl.def("setCacheSize", &WarpingControl::setCacheSize, "cacheSize"_a);
+                clsWarpingControl.def("getInterpLength", &WarpingControl::getInterpLength);
+                clsWarpingControl.def("setInterpLength", &WarpingControl::setInterpLength, "interpLength"_a);
+                clsWarpingControl.def("setWarpingKernelName", &WarpingControl::setWarpingKernelName,
+                                      "warpingKernelName"_a);
+                clsWarpingControl.def("getWarpingKernel", &WarpingControl::getWarpingKernel);
+                clsWarpingControl.def("setWarpingKernel", &WarpingControl::setWarpingKernel, "warpingKernel"_a);
+                clsWarpingControl.def("setMaskWarpingKernelName", &WarpingControl::setMaskWarpingKernelName,
+                                      "maskWarpingKernelName"_a);
+                clsWarpingControl.def("getMaskWarpingKernel", &WarpingControl::getMaskWarpingKernel);
+                clsWarpingControl.def("hasMaskWarpingKernel", &WarpingControl::hasMaskWarpingKernel);
+                clsWarpingControl.def("setMaskWarpingKernelName", &WarpingControl::setMaskWarpingKernelName,
+                                      "maskWarpingKernelName"_a);
+                clsWarpingControl.def("setMaskWarpingKernel", &WarpingControl::setMaskWarpingKernel,
+                                      "maskWarpingKernel"_a);
+                clsWarpingControl.def("getGrowFullMask", &WarpingControl::getGrowFullMask);
+                clsWarpingControl.def("setGrowFullMask", &WarpingControl::setGrowFullMask, "growFullMask"_a);
+            });
+    /* Members */
+}
+} // nameapce
+void wrapWarpExposure(lsst::utils::python::WrapperCollection &wrappers) {
+        declareWarpExposure(wrappers);
```

Comment:
Needs several `addSignatureDependency` to support `Image`, `MaskedImage`, and `Exposure` (probably won't work until you convert `afw.image`), as well as `SkyWcs` (won't work until you convert `afw.geom`).